app/redux/shared_actions/VelocityDropActionCreators.js

```diff
@@ -1,12 +0,0 @@
-import { 
-    VELOCITY_DROPPED, 
-    PLAY_VELOCITY_DROP_AUDIO 
-} from 'app/configs+constants/ActionTypes';
-
-export const velocityDropped = () => ({
-    type: VELOCITY_DROPPED
-});
-
-export const playVelocityDropAudio = () => ({
-    type: PLAY_VELOCITY_DROP_AUDIO
-});
```

Comment:
- [ ] Do we still need these constants?

Reply:
Nope, looks like I just missed them. Removing now.